blog/_posts/2019-11-13-affiliations-feature.md

```diff
@@ -0,0 +1,67 @@
+
+This is too much to work through by hand, but it represents valid affiliation data for millions of author-affiliation pairs. For the unmatched affiliations that occur fewer than 100 times, we’re using machine learning techniques to try and assign ids.  For now, we’re using scikit-learn tools to try and match unknown strings to IDs. Specifically, we’re using scikit-learn’s feature_extraction tools for analyzing the known affiliation strings, and the SGDClassifier to generate models.  We then pass the unknown strings through the model and extract both a best match and a confidence estimate.  The process can be very memory intensive because of the large number of classes -- nearly 6600, one for each aff_id.  A typical data set of 25,000 unmatched strings takes about an hour on a modest workstation (Apple iMac with an Intel i5 processor and 24 GB of memory).  
+
+We’ve found the process is reliable to the limits of the input dictionary, and also found that it’s much more of a data curation problem than a machine learning problem.  Curation of a machine learning model isn’t trivial.  It’s very sensitive to both errors in assigned IDs (which are rare but do occasionally occur) and to ambiguous affiliation strings or strings that can’t be split cleanly.  Examples of the latter include affiliation strings that include (say) both a national laboratory affiliation and a university one (e.g. the Italian INFN centers, NIKHEF member institutions, and US DOE Laboratories), or cases where we have an incomplete mapping of all parent-child relations (e.g. where we have an ID for a university’s Department of Physics, but not a Department of Statistics or Materials Science).
+
+For now, we’re using machine learning to assist with curation, but we’re not yet confident enough to pass it an affiliation string and guarantee it returns the correct ID; it’s not (yet) a part of our hands-off pipeline.  It’s an ongoing process of improvement, and always will be as new metadata keeps coming in.
+
+**What’s next?**
+
+Searching ADS by affiliations is already very useful for helping with disambiguation, and for helping build institutional publication lists.  We hope to make it even more powerful by integrating our system with [ROR](https://ror.org) and integrating publisher-specific identifiers in our workflow.  We are actively working with others to extend ROR identifiers (which are assigned one per institution) to the department level.  In addition, we hope to improve the user experience by coupling affiliations with authors, adding hover-over expansion of abbreviations, and implementing auto-complete with the institution search.  As always, we welcome feedback and corrections.
```

Comment:
* "integrating publisher-specific identifiers in our workflow" And in our searchable fields?
* "We are actively working with other *institutions/projects/whatever* to extend ROR"